docs/config/examples/googlePubSub.md

```diff
@@ -0,0 +1,40 @@
+# Grafana Loki
+
+This example shows how to use `google-pubsub` as a destination for a `watch` data source.
+
+## Configuration
+
+Be sure to have read up on [Manzan configuration](/config/index.md) to understand where these files exist on your system.
+
+### `data.ini`
+
+```ini
+[watchout]
+type=watch
+id=sanjula
+destinations=pubsub_out
+format=$MESSAGE_ID$ (severity $SEVERITY$): $MESSAGE$
+strwch=WCHMSG((*ALL)) WCHMSGQ((*HSTLOG))
+```
+
+### `dests.ini`
+
+```ini
+[pubsub_out]
+type=google-pubsub
+
+# Set your project ID (ie. my-project-438217)
+projectId=<pubsub_project_id>
+
+# Set your topic name (ie. my-topic)
+topicName=<pubsub_topic_name>
+
+# Set your service account name (ie. file:/QOpenSys/etc/manzan/my-project-438217-b7392819a7hf.json)
```

Comment:
A bit confused here as to whether a file path or a key string needs to be specified

Reply:
In Google IAM, you need to create a service account and generate a key for it. This will be a file that the user will put somewhere. You can load the file using `file:` or even `http:`.

More details here: https://camel.apache.org/components/4.8.x/google-pubsub-component.html#_component_option_serviceAccountKey

Reply:
I see. Might be more clear to have in the angle brackets `<path_to_service_account_key>`. To me `<pubsub_service_account_key>` makes it sound like the literal key value should be specified

Reply:
Good idea